conf/test_full.config

```diff
@@ -15,7 +15,7 @@ params {
     config_profile_description = 'Full test dataset to check pipeline function'
 
     // Parameters for full-size test
-    input          = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/v3.1/samplesheet_full.csv'
-    genome         = 'GRCh37'
-    pseudo_aligner = 'salmon'
+    input          = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/v3.4/samplesheet_full.csv'
```

Comment:
Just created https://github.com/nf-core/test-datasets/tree/rnavar
Can you add the samplesheet for that there instead of having it here in the `rnaseq` branch?

Reply:
ok. what should be the version here (e.g. 3.4) for the samplesheet?

Reply:
1.0 for the rnavar coming release :sunglasses:

Reply:
Can you see this: https://github.com/nf-core/test-datasets/pull/527

Reply:
I got confused because you pushed directly to `nf-core/test-datasets@rnavar`, and I merged this PR to `master`.
:scream: 
But reverted quickly and got it back as it should be